spec/tom_queue/tom_queue_integration_spec.rb

```diff
@@ -212,7 +212,7 @@ def start!
 
       # Merge the list of received work together sorted by received time,
       # and compare to the source list
-      sink_order = consumers.map { |c| c.work }.flatten.sort.map { |a| a.payload }
+      sink_order = consumers.map(&:work).flatten.sort { |a, b| a[:run_at] <=> b[:run_at] }.map(&:payload)
```

Comment:
You can also use `Enumerable#sort_by`.

http://ruby-doc.org/core-2.2.2/Enumerable.html#method-i-sort_by